timemachine/cpp/src/nonbonded_all_pairs.cu

```diff
@@ -118,25 +118,36 @@ NonbondedAllPairs<RealType, Interpolated>::NonbondedAllPairs(
     gpuErrchk(cudaMalloc(&d_sort_vals_in_, K_ * sizeof(d_sort_vals_in_)));
 
     // initialize hilbert curve
-    std::vector<unsigned int> bin_to_idx(256 * 256 * 256);
-    for (int i = 0; i < 256; i++) {
-        for (int j = 0; j < 256; j++) {
-            for (int k = 0; k < 256; k++) {
+    const int N_BINS = 128;
+    const int N_BITS = 8;
+    const int capacity = 1 << N_BITS;
```

Comment:
nit: do we have a case convention for constants? E.g. `N_BITS` vs. `capacity`.

Reply:
You're right that was inconsistent -- modified in https://github.com/proteneer/timemachine/pull/692/commits/388e71ed81be993239bf2581ebc4d22067233776 to keep all-caps for the "global" constant defined in `.cuh`, but switched `N_BITS` to `nBits` for the local constant, also to match the parameter name for easier searching later https://github.com/proteneer/timemachine/blob/8b0e4d2281c69b6aba2f62794349fe1e2f7950bf/timemachine/cpp/src/vendored/hilbert.h#L50